draft-documents/portable-reputation/src/README.md

```diff
@@ -0,0 +1,36 @@
+Running:
+
+```sh
+./claim \
+  --target did:bitcoin01:xxx-noah \
+  --claim:summary 'blockchain expert' \
+  --creator did:bitcoin01:xxx-me \
+  --tags Reputation,
+  
+ ```
+ 
+ Should create JSON-LD:
+ 
+ ```js
+ {
+  "@context": "TODO",
+  "id": "did:bitcoin01:xyz123",  // is this valid JSON-LD?
+  "type": ["Claim", "Reputation"],
+  "issuer": "https://portable-reputation-toolkit.xyz",
+  "issued": "2016-10-20T11:36Z",
+  "claim": {
+    "id": "did:bitcoin01:xxx-noah",
+    "summary": "blockchain expert"
+  },
+  "signature": {
+    "type": "sha256-ecdsa-secp256k1-2016",
+    "created": "2016-06-18T21:19:10Z",
+    "creator": "did:bitcoin01:xxx-me",
+    "domain": "TODO",
```

Comment:
`domain` is the audience you'd like to limit the signature to. It can be a website domain, IP address, free form text, or anything else the recipient can use to make sure that the message isn't being replayed from another intended recipient. For example, if you intended a digital signature to only be sent to and used by foobar.com, you'd do: domain: "foobar.com". If baz.com received that signature, it would look at the domain and know that the signature was intended for foobar.com and flag the signature as invalid.